sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+Up close, the texture of the leaf is particularly visible--its ridges and curls, growths and cracks. Behind it, the linoleum tile floor is also thick with texture and grit. Over the course of the first shot, which is the longest of the three, the leaf's texture is amplified and granulated by digital processing. Eventually, it becomes apparent that the leaf is gradually disappearing, the algorithmic process eroding it away to reveal the floor beneath it. Leaf and linoleum, figure and ground, flatten and merge. The leaf eventually blows across the floor to the other side of the frame. Just before its outline fades from view completely, it is swept out of the frame by a sudden gust of wind. After several moments spent in its absence, there is a cut. The leaf returns, seen from another angle. We hear a crackling sound, analogous in quality and rhythm to the leaf's granulation in the previous scene. In this case, though, the digital intervention (the sound) is additive, rather than subtractive. The leaf remains intact. In the final scene, the leaf is once again seen from a different angle. Silent once more, the visual erosion has returned. This time it is apparent that the digital processing is eating away at the entire scene, not only the leaf. More quickly than before, the leaf again fades to nearly nothing and then is caught by the wind and blown out of the frame. The video ends.
+
+![Video still from In Passing, showing a digitally-processed leaf.](../images/video-stills/in-passing.png)
+_Figure x: A video still from_ In Passing.
+
+This description of _In Passing_ highlights several aspects that are relevant to a discussion of its particular embodied technics. First, it is a long, slow, and arguably repetitive video. There is no story to escape into, no self-effacing narrative or expressionistic flow. It takes time and demands attention. Some may call it "difficult art," though I don't find it so. Secondly, the video draws together several forces: the natural energy of the wind and the leaf's hapticity on one hand, and the technological mediation of the camera’s framing and editing along with the modulatory capacity of the digital signal processing algorithm on the other. Once coupled to the embodied perception of the viewer, _In Passing_ forms, yet again, a kind of system in which natural, technical, and bodily energies co-mingle over time. It is an aggregator and modulator of these forces. In this system, the camera serves as a means to amplify the small energies of everyday life, while the processing shifts and expands them. The viewer activates the system, transforming these energies with their eyes, ears, and bodies.
+
+_In Passing_ suggests that a work of art can function differently, perhaps, than one might traditionally expect; as more than just a static object for aesthetic contemplation. As with Jack Chambers' embodied technics of _perceptual realism_, it is a system to see (and hear) with, a means to generate and transform experience.
```

Comment:
Comment From David: "Re: 'embodied technics in In Passing', is this a phrase you’ve already introduced?"